snippets/rpi_5_xen_domd/README.rst

```diff
@@ -0,0 +1,17 @@
+.. _rpi_5_xen_domd:
+
+RPI 5 Xen DomD: snippet for XEN HW domain
```

Comment:
Do we really need separate snippets for DomD and DomUs? AFAIK there is no real difference between them, so what is the reason for explicitly specifying DomD here?

Reply:
if you look at code ypu'll see DomD based on xenvm DT + passedthrough HW, Dom0 has different Xen specific DT and no HW for now. Those are NOT compatible.